apps/meteor/client/views/admin/apps/PriceDisplay.tsx

```diff
@@ -46,13 +46,15 @@ const PriceDisplay: FC<PriceDisplayProps> = ({ purchaseType, pricingPlans, price
 	);
 
 	return (
-		<Box display='flex' flexDirection='column' {...props}>
-			{showType && (
-				<Box color='default' withTruncatedText>
-					{t(type as TranslationKey)}
-				</Box>
-			)}
-			<Box withTruncatedText>{!showType && type === 'Free' ? t(type) : formattedPrice}</Box>
+		<Box mis='x8'>
+			<Tag small>
+				{showType && (
+					<Box color='default' withTruncatedText>
```

Comment:
is this inside Box necessary?

Reply:
Currently yes. The fuselage Tag has a rendering problem/implementation particularity that makes it render any props passed to it directly in the dom which, basically, ignores almost anything given to it outside of its pre-determined props, and because of that I had to use the wrapper Box so that I could implement this margin.

Reply:
This is currently being solved here: https://github.com/RocketChat/fuselage/pull/865